[2025-09-06T14:37:04.935+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db manual__2025-09-04T16:11:37.161994+00:00 [queued]>
[2025-09-06T14:37:04.960+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db manual__2025-09-04T16:11:37.161994+00:00 [queued]>
[2025-09-06T14:37:04.961+0000] {taskinstance.py:2170} INFO - Starting attempt 7 of 8
[2025-09-06T14:37:04.991+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_sql_db> on 2025-09-04 16:11:37.161994+00:00
[2025-09-06T14:37:05.009+0000] {standard_task_runner.py:60} INFO - Started process 620 to run task
[2025-09-06T14:37:05.018+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'banvic_etl', 'extract_sql_db', 'manual__2025-09-04T16:11:37.161994+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/banvic_dag.py', '--cfg-path', '/tmp/tmp80289jva']
[2025-09-06T14:37:05.033+0000] {standard_task_runner.py:88} INFO - Job 67: Subtask extract_sql_db
[2025-09-06T14:37:05.431+0000] {task_command.py:423} INFO - Running <TaskInstance: banvic_etl.extract_sql_db manual__2025-09-04T16:11:37.161994+00:00 [running]> on host 90038f871efe
[2025-09-06T14:37:05.897+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='banvic_etl' AIRFLOW_CTX_TASK_ID='extract_sql_db' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T16:11:37.161994+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-04T16:11:37.161994+00:00'
[2025-09-06T14:37:06.016+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-06T14:37:06.195+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-06T14:37:06.258+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-06T14:37:06.348+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-06T14:37:06.404+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=banvic_etl, task_id=extract_sql_db, execution_date=20250904T161137, start_date=20250906T143704, end_date=20250906T143706
[2025-09-06T14:37:06.489+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-06T14:37:06.555+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
